BridgeAppSDK/SBABridgeManager.m

```diff
@@ -33,15 +33,54 @@
 
 #import "SBABridgeManager.h"
 #import <BridgeAppSDK/BridgeAppSDK-Swift.h>
+@import ResearchUXFactory;
 
 @implementation SBABridgeManager
 
++ (void)addResourceBundleIfNeeded {
+    // Add this bundle to the resource bundles
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        SBAInfoManager *infoManager = [SBAInfoManager sharedManager];
+        NSMutableArray *bundles = [infoManager.resourceBundles mutableCopy];
+        NSBundle *thisBundle = [NSBundle bundleForClass:[self class]];
+        if (![bundles containsObject:thisBundle]) {
+            NSUInteger idx = bundles.count > 0 ? 1 : 0;
+            [bundles insertObject:thisBundle atIndex:idx];
+            infoManager.resourceBundles = bundles;
+        }
+    });
+}
+
++ (void)setupWithBridgeInfo:(id <SBBBridgeInfoProtocol>)bridgeInfo
+                participant:(id <SBAParticipantInfo>)participant
+               authDelegate:(id <SBBAuthManagerDelegateProtocol>) authDelegate {
+    
+    // Add the resource bundle (if needed)
+    [self addResourceBundleIfNeeded];
+    
+    // Set the participant
+    [SBAInfoManager sharedManager].currentParticipant = participant;
+    
+    // Setup the Bridge study
+    [BridgeSDK setupWithBridgeInfo:bridgeInfo];
+    [SBBComponent(SBBAuthManager) setAuthDelegate:authDelegate];
+}
+
 + (void)setupWithStudy:(NSString *)study
         cacheDaysAhead:(NSInteger)cacheDaysAhead
        cacheDaysBehind:(NSInteger)cacheDaysBehind
            environment:(SBBEnvironment)environment
           authDelegate:(id <SBBAuthManagerDelegateProtocol>) authDelegate {
     
+    [self addResourceBundleIfNeeded];
+    
+    // If the auth delegate is also the current user then set that
+    if ([authDelegate conformsToProtocol:@protocol(SBAParticipantInfo)]) {
```

Comment:
This is a deprecated method but it's also the method wedMD is accessing directly b/c they already have an app where the delegate is obj-c so it can't inherit from `SBAAppDelegate`.

Basically, once I'd pulled out the factory into its own library, I needed a way to point at the `SBAParticipantInfo`. Figured this was best approach since then webMD would still work.